### ISSUE TITLE
Cannot change objective of already solved model

### ISSUE THREAD

**Issue by stelmo** [oxfordcontrol/Clarabel.jl#99]
Hi, for an application I am developing I need to solve an LP and QP that are very similar. My current way of doing this is to just change the objective of the JuMP model, but I cannot do this using Clarabel (with other solvers this works fine). Is there a workaround?
```julia
using JuMP #v1.2.1
using Clarabel #v0.2.0  

m = Model(Clarabel.Optimizer)
x = @variable(m, x[1:2])
@objective(m, Max, x[2])
@constraint(m, x[1] + x[2] == 2)
@constraint(m, x[2] <= 3)
@constraint(m, x[1] <= 3)
optimize!(m)

@objective(m, Min, sum(x .^ 2))
```
This throws:
```julia
ERROR: MathOptInterface.UnsupportedAttribute{MathOptInterface.ObjectiveSense}: Attribute MathOptInterface.ObjectiveSense() is not supported by the model.
Stacktrace:
  [1] throw_set_error_fallback(model::Optimizer{Float64}, attr::MathOptInterface.ObjectiveSense, value::MathOptInterface.OptimizationSense; error_if_supported::MathOptInterface.SetAttributeNotAllowed{MathOptInterface.ObjectiveSense})
    @ MathOptInterface C:\Users\stelmo\.julia\packages\MathOptInterface\MMUy6\src\attributes.jl:528
  [2] throw_set_error_fallback(model::Optimizer{Float64}, attr::MathOptInterface.ObjectiveSense, value::MathOptInterface.OptimizationSense)
    @ MathOptInterface C:\Users\stelmo\.julia\packages\MathOptInterface\MMUy6\src\attributes.jl:519
  [3] set(model::Optimizer{Float64}, attr::MathOptInterface.ObjectiveSense, args::MathOptInterface.OptimizationSense)
    @ MathOptInterface C:\Users\stelmo\.julia\packages\MathOptInterface\MMUy6\src\attributes.jl:513
  [4] set(m::MathOptInterface.Utilities.CachingOptimizer{Optimizer{Float64}, MathOptInterface.Utilities.UniversalFallback{MathOptInterface.Utilities.Model{Float64}}}, attr::MathOptInterface.ObjectiveSense, value::MathOptInterface.OptimizationSense)
    @ MathOptInterface.Utilities C:\Users\stelmo\.julia\packages\MathOptInterface\MMUy6\src\Utilities\cachingoptimizer.jl:764
  [5] set(b::MathOptInterface.Bridges.LazyBridgeOptimizer{MathOptInterface.Utilities.CachingOptimizer{Optimizer{Float64}, MathOptInterface.Utilities.UniversalFallback{MathOptInterface.Utilities.Model{Float64}}}}, attr::MathOptInterface.ObjectiveSense, value::MathOptInterface.OptimizationSense)
    @ MathOptInterface.Bridges C:\Users\stelmo\.julia\packages\MathOptInterface\MMUy6\src\Bridges\bridge_optimizer.jl:1013
  [6] set(m::MathOptInterface.Utilities.CachingOptimizer{MathOptInterface.Bridges.LazyBridgeOptimizer{MathOptInterface.Utilities.CachingOptimizer{Optimizer{Float64}, MathOptInterface.Utilities.UniversalFallback{MathOptInterface.Utilities.Model{Float64}}}}, MathOptInterface.Utilities.UniversalFallback{MathOptInterface.Utilities.Model{Float64}}}, attr::MathOptInterface.ObjectiveSense, value::MathOptInterface.OptimizationSense)
    @ MathOptInterface.Utilities C:\Users\stelmo\.julia\packages\MathOptInterface\MMUy6\src\Utilities\cachingoptimizer.jl:764
  [7] set(m::Model, attr::MathOptInterface.ObjectiveSense, value::MathOptInterface.OptimizationSense)
    @ JuMP C:\Users\stelmo\.julia\packages\JuMP\60Bnj\src\JuMP.jl:1236
  [8] set_objective_sense
    @ C:\Users\stelmo\.julia\packages\JuMP\60Bnj\src\objective.jl:78 [inlined]
  [9] set_objective(model::Model, sense::MathOptInterface.OptimizationSense, func::QuadExpr)
    @ JuMP C:\Users\stelmo\.julia\packages\JuMP\60Bnj\src\objective.jl:151
 [10] macro expansion
    @ C:\Users\stelmo\.julia\packages\JuMP\60Bnj\src\macros.jl:1288 [inlined]
 [11] top-level scope
    @ REPL[14]:1
```

**Comment by goulart-paul** [comment]
This appears (?) to already be fixed in our v0.3.0 development version, which we expect to release fairly soon.    I will ping you when it is available.

**Comment by goulart-paul** [comment]
@stelmo We have released v0.3.0 of this solver.   Let us know if it does not fix the issue for you.

**Comment by stelmo** [comment]
All our tests pass! Thank you very much for keeping us in the loop :)
